source/guide_postgresql.rst

```diff
@@ -298,14 +350,96 @@ Create Database
  * ``--template``: PostgreSQL supports standard templates to create the database structure.
  * ``database name``: And as last option the name of the database. In this example 'synapse'.
 
-.. warning:: Please replace ``<user>`` with the user name, created earlier!
+.. warning:: Please replace ``<user>`` with the user name, created earlier, and <database name> with the name of the database you want to create!
 
 .. code-block:: console
  :emphasize-lines: 1
 
  [isabell@stardust ~]$ createdb --encoding=UTF8 --owner=<user> --template=template0 synapse
  [isabell@stardust ~]$
 
+PostgreSQL Extensions
+=====================
+
+PostgreSQL `extensions <https://wiki.postgresql.org/wiki/Extensions>`_ allow to extend the functionality and user-visible functions of a database. A number of extensions for PostgreSQL are available (see `list <https://www.postgresql.org/download/products/6-postgresql-extensions/>`_), with the geospatial extension PostGIS_ being one of the most widely used ones.
+
+Available extensions for PostgreSQL on U7 can be found in the ``/usr/pgsql-<MajorVersion>/share/extension`` directory, where ``<MajorVersion>`` refers to the PostgreSQL `Version`_ of your installation.
+
+.. Note:: In order to create extensions, the PostgreSQL user will need to have administrator rights. These are automatically assigned to the database owner using the ``--owner`` flag, as shown in the `Create database` section.
+
+
+PostGIS: Spatially enabling the database using PostGIS
+----------------------------------------------------------
+
+PostGIS_ adds support for geographic objects to the database, allowing to run spatial
+queries. The PostgreSQL installation on Uberspace comes with pre-compiled versions
+of the PostGIS extension saving you from having to compile PostGIS and its FOSS GIS software stack dependencies (such as `GDAL <https://gdal.org>`_, `PROJ <http://proj.org>`_, `GEOS <https://trac.osgeo.org/geos/>`_ and `SFCGAL <http://www.sfcgal.org/>`_) from source.
+
+To check whether PostGIS_ exists for the PostgreSQL version do a quick ``ls`` of the ``share``
+directory. In this guide we use PostGIS_ version 3.1 with a PostgreSQL major version 12:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ ls -rtl /usr/pgsql-12/share/extension/postgis--3*.sql
+ [isabell@stardust ~]$ -rw-r--r--. 1 root root 7.9M May 26 15:59 /usr/pgsql-12/share/extension/postgis--3.1.2.sql
+
+
+Once you have convinced yourself that the right PostGIS_ extension is available, simply enter the database which you have just created (`Create Database`_) as administrator and spatially `enable the database <http://postgis.net/docs/postgis_administration.html#create_spatial_db>`_ using the PostGIS extension:
+
+.. warning:: Please replace ``<user>`` with the user name, created earlier, and ``<database name>`` with the name of the database you want to create!
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ psql <database name> -U<user>
```

Comment:
I thinke we should prefer _long options_:
```suggestion
[isabell@stardust ~]$ psql <database name> --username <user>
```

Reply:
This is probably not needed anymore as PostgreSQL automatically uses the unix login name (in our case the Uberspace account name) when no `--username` is provided. For installing extensions this means that we're directly in superuser mode and can install extensions. This is clarified in the new version of the guide.